docs/learn/uses-of-tezos/nft.md

```diff
@@ -24,23 +24,22 @@ Recently, NFTs have been a powerful primitive demonstrating usefulness in the cr
 * [tzNFT](https://github.com/tqtezos/nft-tutorial) - a tutorial showing users how to originate and interact with the FA2 NFT contract implementation.
 * [TZIP-021](https://gitlab.com/tzip/tzip/-/blob/tzip-21-spec/proposals/tzip-21/tzip-21.md) - an emerging contract multimedia metadata standard. 
 
-### Tools and Applications
-
-* \*\*\*\*[Truesy](https://www.truesy.com/) ****- a highly curated NFT marketplace
-* [Kalamint](https://kalamint.io/) - A community owned NFT marketplace on Tezos.
-* [OpenMinter](https://github.com/tqtezos/minter) ****and [Minter-SDK](https://github.com/tqtezos/minter-sdk) - A reusable dApp and SDK that allows anyone to create, collect, sell, or swap NFTs on Tezos.
-* \*\*\*\*[tzcolors](https://www.tzcolors.io)\*\*\*\*
-* \*\*\*\*[TezAuction](https://tez.auction/#/)\*\*\*\*
-* [CricTez](https://crictez.medium.com/introducing-crictez-21a284c60e94)
+### NFT Marketplaces 
+* [Objkt](https://objkt.com/) - The largest NFT marketplace on Tezos
+* [Kalamint](https://kalamint.io/) - A clean NFT marketplace on the Tezos blockchain
+* [Hic Et Nunc](https://hicetnunc.art/) **- The site hicetnunc.xyz was discontinued, and replaced by the community with hicetnunc.art**
+* [tzcolors](https://www.tzcolors.io) - Unique colors as NFT tokens on Tezos
 * [MinterPop](https://minterpop.com/)
-* [OneOf](https://oneof.com/)
-* [RedBull Racing Honda](https://www.redbull.com/int-en/redbullracing/tezos-joins-as-official-blockchain-partner)
-* [McLaren Racing](https://www.mclaren.com/racing/partners/tezos/)
-* [Emergents TCG](https://www.emergents.gg/)
-* [Play with Brio ](https://playwithbrio.com/home)
-* [Tezos Profiles](https://tzprofiles.com/)
-* [Ipocamp](https://www.ipocamp.io/)
-
+* [OneOf](https://oneof.com/) - OneOf is a green NFT platform built for the music community to connect fans and collectors at all levels with their favorite artists.
+* [RedBull Racing Honda](https://redbullracingcollectibles.com/) - Built on Tezos, our blockchain partner, we’re bringing the world of Red Bull Racing closer to our fans with a series of amazing digital artifacts.
+* [McLaren Racing](https://mclarenracingcollective.com/) - The McLaren Racing Collective, powered by Tezos, is an innovative platform where fans and collectors can purchase McLaren Racing digital collectibles or non-fungible tokens (NFTs).
+* [Emergents TCG](https://emergentstcg.com/) - The trading card game where you own your cards
+* [Play with Brio ](https://playwithbrio.com/home) - Welcome to the first ever eSports skill based platform on the Tezos blockchain
```

Comment:
Please edit description to "The first ever eSports skill based platform on the Tezos blockchain"